Source/Core/DiscIO/DiscExtractor.cpp

```diff
@@ -325,8 +325,8 @@ bool ExportDOL(const Volume& volume, const Partition& partition, const std::stri
 std::optional<u64> GetFSTOffset(const Volume& volume, const Partition& partition)
 {
   const Platform volume_type = volume.GetVolumeType();
-  if (!IsDisc(volume_type))
-    return {};
+  if (IsDisc(volume_type))
```

Comment:
Is removing the unary not here intentional?

Reply:
no. typo. good catch